### PR TITLE
Update preview.md

### PR DESCRIPTION
**Pull request by thibaultchatiron**
Integration with MCAS has been backported on Windows 10 v1709

### CODE REVIEW

**Comment by e0i** [comment]
@mjcaparas The PR that introduces a minor version update is available for review. Thanks.

**Comment by mjcaparas** [comment]
@e0i - please see my comment on the required update on the text thank you

**Comment by e0i** [comment]
@thibaultchatiron Please follow-up with suggested changes so that the PR can proceed into merge. Thanks.



> Please update with this text instead:
> 
> This feature will be available with an E5 license for Enterprise Mobility + Security on machines running Windows 10, version 1709 (OS Build 16299.1085 with KB4493441), Windows 10, version 1803 (OS Build 17134.704 with KB4493464), Windows 10, version 1809 (OS Build 17763.379 with KB4489899) or later Windows 10 versions.

File: windows/security/threat-protection/microsoft-defender-atp/preview.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Information protection is an integral part of Microsoft 365 Enterprise suite, pr
 - [Integration with Microsoft Cloud App Security](microsoft-cloud-app-security-integration.md) <BR> Microsoft Cloud App Security leverages Microsoft Defender ATP endpoint signals to allow direct visibility into cloud application usage including the use of unsupported cloud services (shadow IT) from all Microsoft Defender ATP monitored machines.
 
     >[!NOTE]
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/preview.md]: recommended MarkDown standard:
```suggestion
    > [!NOTE]
```

File: windows/security/threat-protection/microsoft-defender-atp/preview.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ Information protection is an integral part of Microsoft 365 Enterprise suite, pr
...
-    >Available from Windows 10, version 1809 or later.
+    >Available from Windows 10, version 1709 or later.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/preview.md]:
```suggestion
    > This feature will be available with an E5 license for Enterprise Mobility + Security on machines running Windows 10, version 1709 (OS Build 16299.1085 with KB4493441), Windows 10, version 1803 (OS Build 17134.704 with KB4493464), Windows 10, version 1809 (OS Build 17763.379 with KB4489899) or later Windows 10 versions.
```

**Comment by e0i** [comment]
@thibaultchatiron Any updates with revision suggestions provided by **illfated** and **mjcaparas** ? We would like to merge your contribution. Thanks.

**Comment by illfated** [comment]
Maybe @thibaultchatiron is on vacation. It looks like he hasn't been doing anything on Github the last week. I guess we will just have to wait for a few more days until he is back.

**Comment by e0i** [comment]
@thibaultchatiron Following up for updates.

**Comment by e0i** [comment]
@thibaultchatiron Closing PR as there has been no response regarding revisions. Feel free to re-submit the PR so that your contributions can be merged. Thanks.

**Comment by e0i** [comment]
@officedocsbot close

**Pull request closed without merging**